scio-extra/src/main/scala/com/spotify/scio/extra/transforms/package.scala

```diff
@@ -112,4 +116,42 @@ package object transforms {
 
   }
 
+  /**
```

Comment:
Sure, though I did copy this from above. It seems that all implicit classes use scalaDoc